app/api/search/entitiesIndex.js

```diff
@@ -38,6 +35,7 @@ function setFullTextSettings(defaultDocument, id, body, doc) {
 
 const bulkIndex = async (docs, _action = 'index', elasticIndex) => {
   const body = [];
+  // eslint-disable-next-line max-statements
```

Comment:
I have not changed the following .forEach nor appended any code to it.  We should probably refactor this.